scripts/testing/estimate_calories_from_image.py

```diff
@@ -0,0 +1,318 @@
+
+from PIL import Image
+import numpy as np
+import cv2 as cv
+import math
+
+
+COIN_WIDTH = 2.5
+
+FOOD_LABELS = {
+    'apple',
+    'banana',
+    'bread',
+    'bun',
+    'doughnut'
+    'egg'
+    'fried dough twist',
+    'grape',
+    'lemon',
+    'litchi',
+    'mango',
+    'mooncake',
+    'orange',
+    'peach',
+    'pear',
+    'plum',
+    'qiwi',
+    'sachima',
+    'tomato',
+}
+
+# value can be 'ellipsoid', 'column', 'irregular'
+FOOD_SHAPE = {
+  'apple' : 'ellipsoid',
+  'banana' : 'irregular',
+  'bread' : 'column',
+  'bun' : 'irregular',
+  'doughnut' : 'irregular',
+  'egg' : 'ellipsoid',
+  'fried dough twist' : 'irregular',
+  'grape' : 'column',
+  'lemon' : 'ellipsoid',
+  'litchi' : 'irregular',
+  'mango' : 'irregular',
+  'mooncake' : 'column',
+  'orange' : 'ellipsoid',
+  'peach' : 'ellipsoid',
+  'pear' : 'irregular',
+  'plum' : 'ellipsoid',
+  'qiwi' : 'ellipsoid',
+  'sachima' : 'column',
+  'tomato' : 'ellipsoid'
+}
+
+# TODO: Populate these values with empirical data that we come up with
+FOOD_BETA = {
+  'apple' : 1.0,
+  'banana' : 1.0,
+  'bread' : 1.0,
+  'bun' : 1.0,
+  'doughnut' : 1.0,
+  'egg' : 1.0,
+  'fried dough twist' : 1.0,
+  'grape' : 1.0,
+  'lemon' : 1.0,
+  'litchi' : 1.0,
+  'mango' : 1.0,
+  'mooncake' : 1.0,
+  'orange' : 1.0,
+  'peach' : 1.0,
+  'pear' : 1.0,
+  'plum' : 1.0,
+  'qiwi' : 1.0,
+  'sachima' : 1.0,
+  'tomato' : 1.0
+}
+
+FOOD_CALORIE = { 
```

Comment:
Can you leave a little comment that tells us what this value is? Or what the unit for this mapping value is?